examples/simple.rs

```diff
@@ -1,12 +1,29 @@
 extern crate oping;
-use oping::{Ping, PingResult};
+use oping::{Ping, PingResult, PingError};
 
 fn do_stuff() -> PingResult<()> {
     let mut ping = Ping::new();
     try!(ping.set_timeout(5.0));  // timeout of 5.0 seconds
     try!(ping.add_host("localhost"));  // fails here if socket can't be created
     try!(ping.add_host("::1"));  // IPv4 / IPv6 addresses OK
-    try!(ping.add_host("1.2.3.4"));
+    
+    /* Replacing the following 'try' macro with an example of how to use
+     * my PingError::CustomError. I suggest adding a broken IP Address like
+     * 1.300.1.1 to see it in action.
+     *      - Timmonfette1
+     *
+     * try!(ping.add_host("1.2.3.4"));
+     */
+    let ip = "1.2.3.4";
+    match ping.add_host(ip) {
```

Comment:
A `.map_err(|e| ...)` on the `Result` would be more idiomatic than a match here, FWIW -- and if we don't do that, then the `return` in mid-expression is somewhat awkward too (sort of a symmetry thing -- better to let the `match` always evaluate, then wrap it in a `try!(...)`).